python/foglamp/services/core/api/utils.py

```diff
@@ -16,6 +16,7 @@ def get_plugin_info(name):
         out, err = p.communicate()
         res = out.decode("utf-8")
         jdoc = json.loads(res)
+        _logger.error("C Plugin config: ", res)
```

Comment:
This should not be an error, I would suggest use `_logger.debug("C Plugin config: %s", res)`

Reply:
That was debug that got left accidentally, will remove